statusnet/statusnet.php

```diff
@@ -1050,7 +1050,7 @@ function statusnet_createpost(App $a, $uid, $post, $self, $create_user, $only_ex
 		$own_url = DI::pConfig()->get($uid, 'statusnet', 'own_url');
 
 		if ($content->user->id == $own_url) {
-			$self = DBA::selectFirst('', ['self' => true, 'uid' => $uid]);
+			$self = DBA::selectFirst('*', ['self' => true, 'uid' => $uid]);
```

Comment:
No, I mean, it needs a table name as first argument, and then an array of field names which can be empty, meaning `*`.
```suggestion
			$self = DBA::selectFirst('table', [], ['self' => true, 'uid' => $uid]);
```
Not sure which table it is though but it looks like `contact`.